### ISSUE TITLE
Unable to change val dataloader batch size in qat training

### ISSUE THREAD

**Issue by haritsahm** [Deci-AI/super-gradients#1115]
I've been working all day trying to figure out how to export qat model for different batch size. So from what I understand, I just need to override `dataset_params.val_dataloader_params.batch_size` so that I can use different batch size to export the onnx model as defined in [sg_trainer.py#L2250](https://github.com/Deci-AI/super-gradients/blob/aed7e96aac5794feef89c4bdafb1d36016312d9f/src/super_gradients/training/sg_trainer/sg_trainer.py#L2250).

After a while, I know that it uses the same `batch_size` as the `train_dataloader_params.batch_size` after applying `pre_launch_callback` function because the `val_dataloader_params` uses/copy `train_dataloader_params` in [pre_launch_callbacks.py#L397](https://github.com/Deci-AI/super-gradients/blob/aed7e96aac5794feef89c4bdafb1d36016312d9f/src/super_gradients/training/pre_launch_callbacks/pre_launch_callbacks.py#L397)

Is there any particular reason why you override it with `train_dataloader_params`?

**Comment by spsancti** [comment]
Looks like a copy/paste error to me
@shaydeci @Louis-Dupont do you know it it has some special meaning here?

**Comment by shaydeci** [comment]
Nope, typo. Fixed in https://github.com/Deci-AI/super-gradients/pull/1129 and will be included in the upcoming release.
